src/core/thread/mle.cpp

```diff
@@ -788,15 +788,20 @@ ThreadError Mle::AppendLeaderData(Message &aMessage)
     return aMessage.Append(&mLeaderData, sizeof(mLeaderData));
 }
 
+void Mle::FillNetworkDataTlv(NetworkDataTlv &aTlv, bool aStableOnly)
```

Comment:
How about we modify `AppendNetworkData()` to take a bool argument that indicates whether or not the the TLV is for MLE or diag?